js/main.js

```diff
@@ -1,3 +1,25 @@
-import './inactivation-interface.js';
-import './fill-map.js';
+import {disableInterface, enableInterface} from './activation-interface.js';
+import {mapLoaded, mainPin} from './libs/leaflet-init.js';
 import './validate-form.js';
+import {watchFormDataSubmit} from './form.js';
+import {getData} from './api.js';
+import {markerMoveend} from './utils.js';
+import {renderAdds} from './fill-map.js';
+import {showDataError} from './show-error.js';
+
+// Деактивировали интерфейс
+disableInterface();
+
+// Инициализировали карту
+if (mapLoaded) {
+  enableInterface();
+}
+
+// Получаем данные с сервера, отрисовываем метки и обрабатываем ошибку загрузки
+getData(renderAdds, showDataError);
+
+// Взаимодействуем с главным пином
+mainPin.on('moveend', markerMoveend);
```

Comment:
https://up.htmlacademy.ru/javascript/25/criteries#d4

```suggestion
mainPin.on('moveend', onMainPinMoveend);
```